src/Symfony/Bundle/FrameworkBundle/Command/WorkflowDumpCommand.php

```diff
@@ -39,13 +41,23 @@ protected function configure()
                 new InputArgument('name', InputArgument::REQUIRED, 'A workflow name'),
                 new InputArgument('marking', InputArgument::IS_ARRAY, 'A marking (a list of places)'),
                 new InputOption('label', 'l', InputArgument::OPTIONAL, 'Labels a graph'),
+                new InputOption('format', null, InputArgument::OPTIONAL, 'The dump format', 'dot'),
```

Comment:
"The dump format [dot|puml]"

Reply:
format is a bad name because it's already used in the --help command.
So `bin/console  workflow:dump article --format=puml -h ` do not work anymore

Reply:
good catch I have renamed by `dump_format`